style.css

```diff
@@ -35,6 +42,7 @@
   color: #6070ff;
   padding: 4px;
   margin-top: 58px;
+  margin-left: 24px;
 }
 
 .hambuger {
```

Comment:
- [ ] I noticed that there is some additional gap at the top of the navbar menu please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734205-81725603-0e0b-4b22-ae72-b5a73e6397cc.png)

In order to match it with the original design, I would recommend you loose the fixed margin-top value of 58px both on the logo and hamburger 
figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734293-7a3beedb-92ff-45e0-8ec3-9c61d1ee2dd0.png)

Reply:
Great job working on the previous reviewer's suggestions! 🚀 However, please make sure you follow as closely as possible the Figma design. At the moment, this margin-top problem hasn't been addressed yet:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/230743814-6705b9a1-41e2-4327-82da-8cc8568518b3.png) | ![image](https://user-images.githubusercontent.com/77069011/230743787-40ac66e3-aa04-4df8-bcc4-31511f05bad2.png) |

- [ ] This happens because you have a `margin-top: 58px` property in both, the `.logo` and `.hamburger` elements, please consider removing them! 🤓 🔧 
- [ ] **[OPTIONAL]** Also, if you look closely, the elements in the design have horizontal spacing from the edges, but yours don't, I strongly suggest you fix this! 🤓 🔧